packages/create-plugin/templates/app/src/components/AppConfig/AppConfig.tsx

```diff
@@ -50,6 +50,7 @@ export const AppConfig = ({ plugin }: AppConfigProps) => {
         <Field label="API Key" description="A secret key for authenticating to our custom API">
           <SecretInput
             width={60}
+            id="config-api-key"
```

Comment:
..so that input field becomes associated with the label field.

Reply:
For my own understanding - is this required for playwright to find the input field?

Reply:
You can find the input field using test id too, but it's preferred to test user-visible behavior if possible. So `await page.getByRole('textbox', { name: 'API Key' })` is preferred over `await page.getByTestId('config-api-key').`

Ofc this is not always possible. When accessing grafana ui elements outside of the plugin code, plugin-e2e users should use the e2e selectors. e.g `panelEditPage.getByGrafanaSelector(selectors.components.CodeEditor.container)`. There's more info regarding this in the [Select UI elements](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/selecting-elements) guide in the e2e docs.